frontend/talentsearch/src/js/components/reviewMyApplication/ReviewMyApplicationPage.tsx

```diff
@@ -0,0 +1,266 @@
+import * as React from "react";
+import { useIntl } from "react-intl";
+import UserProfile from "@common/components/UserProfile";
+import Pending from "@common/components/Pending";
+import NotFound from "@common/components/NotFound";
+import { commonMessages } from "@common/messages";
+import { getLocale } from "@common/helpers/localize";
+import MissingSkills from "@common/components/skills/MissingSkills";
+import { notEmpty } from "@common/helpers/util";
+import ExperienceSection from "@common/components/UserProfile/ExperienceSection";
+import { Link } from "@common/components";
+import { getMissingSkills } from "@common/components/skills/MissingSkills/MissingSkills";
+import {
+  Applicant,
+  Maybe,
+  PoolAdvertisement,
+  useGetReviewMyApplicationPageDataQuery,
+} from "../../api/generated";
+import { useDirectIntakeRoutes } from "../../directIntakeRoutes";
+import ApplicationPageWrapper from "../ApplicationPageWrapper/ApplicationPageWrapper";
+import { flattenExperienceSkills } from "../experienceAndSkills/ExperienceAndSkills";
+
+interface ReviewMyApplicationProps {
+  applicant: Applicant;
+  poolAdvertisement: PoolAdvertisement;
+  poolCandidateId: string;
+  closingDate: Date;
+  jobTitle: Maybe<string>;
+  poolId: string;
+}
+
+export const ReviewMyApplication: React.FunctionComponent<
+  ReviewMyApplicationProps
+> = ({
+  applicant,
+  poolAdvertisement,
+  poolCandidateId,
+  closingDate,
+  jobTitle,
+  poolId,
+}) => {
+  const intl = useIntl();
+  const directIntakePaths = useDirectIntakeRoutes();
+  const experiences = applicant.experiences?.filter(notEmpty) || [];
+  const missingSkills = {
+    requiredSkills: poolAdvertisement.essentialSkills?.filter(notEmpty),
+    optionalSkills: poolAdvertisement.nonessentialSkills?.filter(notEmpty),
+  };
+  const hasExperiences = notEmpty(applicant.experiences);
+  const { isProfileComplete } = applicant;
+  const isApplicationComplete =
+    isProfileComplete ||
+    getMissingSkills(
+      missingSkills.requiredSkills || [],
+      hasExperiences ? flattenExperienceSkills(experiences) : [],
+    ).length === 0;
+
+  return (
+    <ApplicationPageWrapper
```

Comment:
I know this is out of scope for this PR so I'll leave it up to you to tag it on or spin it off. :smile:  The titles in the wrapper should probably have drop shadows with the new h2 look. 
![image](https://user-images.githubusercontent.com/8978655/187977062-e732f125-d5af-4ada-b9b4-fadf44330632.png)

Reply:
Fixed by [Improve styling to match designs](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3796/commits/4c06217b89c30bf6548078c889c46d6de10ca560)